medusa/server/web/home/add_shows.py

```diff
@@ -297,7 +297,7 @@ def existingShows(self):
     def addShowByID(self, indexername=None, seriesid=None, show_name=None, which_series=None,
                     indexer_lang=None, root_dir=None, default_status=None,
                     quality_preset=None, any_qualities=None, best_qualities=None,
-                    flatten_folders=None, subtitles=None, full_show_path=None,
+                    season_folders=None, subtitles=None, full_show_path=None,
                     other_shows=None, skip_show=None, provided_indexer=None,
                     anime=None, scene=None, blacklist=None, whitelist=None,
                     default_status_after=None, default_flatten_folders=None,
```

Comment:
Remove `default_flatten_folders` - it isn't used.
Should also change these lines:
https://github.com/pymedusa/Medusa/blob/2a35e2405272bab897b349c23f0787450d65f5c0/themes-default/slim/static/js/add-shows/init.js#L140-L141
to: (remove the comment line)
```js
season_folders: $('#season_folders').prop('checked'), // eslint-disable-line camelcase
```

This should actually fix #4170...